packages/server/graphql/private/mutations/runScheduledJobs.ts

```diff
@@ -60,15 +63,17 @@ const runScheduledJobs: MutationResolvers['runScheduledJobs'] = async (
   {seconds},
   {dataLoader}
 ) => {
-  const r = await getRethink()
+  const pg = getKysely()
   const now = new Date()
 
   // RESOLUTION
   const before = new Date(now.getTime() + seconds * 1000)
-  const upcomingJobs = (await r
-    .table('ScheduledJob')
-    .between(r.minval, before, {index: 'runAt'})
-    .run()) as ScheduledJobUnion[]
+  const upcomingJobs = await pg
+    .selectFrom('ScheduledJob')
+    .selectAll()
+    .where('runAt', '>=', new Date(0))
```

Comment:
+1 I think this line could just be removed

Reply:
Yeah, that's probably true. I was being a bit too direct in my translation ;)